extensions/CocoStudio/Reader/SceneReader.h

```diff
@@ -62,8 +62,11 @@ class CC_EX_DLL SceneReader
 {
 
 public:
-	static SceneReader* sharedSceneReader();
-	static void purge();
+	CC_DEPRECATED_ATTRIBUTE static SceneReader* sharedSceneReader();
+	static SceneReader* getInstance();
+	CC_DEPRECATED_ATTRIBUTE static void purge();
+	static void destroyInstance();
+	static void destroyCocosStudio();
```

Comment:
Jianhua suggest add such a function to cleanup all cocos reader resource, and I think this name is more clear to a global clean up function. SceneReader::destroyInstance is used to clean up SceneReader itself